modelci/hub/manager.py

```diff
@@ -405,3 +411,23 @@ def retrieve_model_by_task(task: Task) -> List[ModelBO]:
     _get_remote_model_weights(models)
 
     return models
+
+
+def retrieve_model_by_parent_id(parent_id: str) -> List[ModelBO]:
+    """
+    Query models by specifying the parent model id
+
+    Args:
+        parent_id (str): : the parent model id of current model if this model is derived from a pre-existing one
+
+    Returns:
+        List[ModelBO]: A list of model business object.
+    """
+    models = ModelService.get_models_by_parent_id(parent_id)
+    # check if found
+    if len(models) == 0:
+        raise FileNotFoundError('Model not found!')
+
+    _get_remote_model_weights(models)
```

Comment:
When you get the child models, do you really need to get them downloaded?

Reply:
No need and the model object should be returned